tests/test_all_72_chat_combinations.py

```
@@ -0,0 +1,328 @@
+            
+            # その他のエラーの場合は詳細を記録
+            if init_response.status_code != 200:
+                print(f"❌ 組み合わせ{combination_index + 1}: 初期化失敗")
+                print(f"   ステータス: {init_response.status_code}")
+                print(f"   レスポンス: {init_response.data.decode('utf-8')[:200]}")
+                
+                # 503 (Service Unavailable) などの場合もスキップ
+                if init_response.status_code in [503, 502, 500]:
+                    pytest.skip(f"サービス一時利用不可により組み合わせ{combination_index + 1}をスキップ")
+                
+                assert False, f"組み合わせ{combination_index + 1}の初期化失敗"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)